src/core/api/user.go

```diff
@@ -151,7 +151,9 @@ func (ua *UserAPI) Get() {
 		}
 		u.Password = ""
 		if ua.userID == ua.currentUserID {
-			u.HasAdminRole = ua.SecurityCtx.IsSysAdmin()
+			// Because UI is rendered with the condition of HasAdminRole,
+			// copy admin privilege to this field to REST response for UI to render it correctly
+			u.SysAdminFlag = ua.SecurityCtx.IsSysAdmin()
```

Comment:
This is wrong, the systemAdminFlag should only reflects the column in DB, but `IsSysAdmin` is the merged result.